tests/test_model.py

```diff
@@ -248,7 +248,12 @@ def test_basic_decoding() -> None:
     '"created": null, '
     '"content": {'
     '"parts": ['
-    '{"mime_type": null, "image_url": "https://miquido.com/image", "image_description": null}'
+    "{"
```

Comment:
Why all those test data are not fixtures?

Reply:
Right now those objects are module level. They can be mutated putting other tests at risk. Do we use random test order?

Reply:
Those tests are yet to be refined and rewritten 😇

Reply:
All of our data models are immutable - there is no risk of making accidental change there